templates/Gemfile.erb

```diff
@@ -27,9 +27,10 @@ gem "fluent-plugin-rewrite-tag-filter", "~> 2.2.0"
 gem "elasticsearch", "~> 6.0"
 gem "fluent-plugin-elasticsearch", "~> 3.5.5"
 <% when "elasticsearch7" %>
-gem "elasticsearch", "~> 7.0"
-gem "fluent-plugin-elasticsearch", "~> 3.7.1"
+gem "fluent-plugin-dedot_filter", "~> 1.0"
+gem "elasticsearch", "~> 7.4.0"
 gem "elasticsearch-xpack", "~> 7.4.0"
+gem "fluent-plugin-elasticsearch", "~> 4.0"
```

Comment:
ditto

Reply:
this is the latest release available https://rubygems.org/gems/fluent-plugin-elasticsearch

Reply:
Yes. but this PR is for installing dedot plugin. Updating elasticsearch is not related unless dedot plugin needs the version of elasticsearch.